lib/users-list.js

```diff
@@ -39,7 +39,7 @@ function updateUserList(ops) {
       log('info', `User data updates for user: ${result._id}`);
       return result;
     }).catch(function (error) {
-      log('error', error.stack);
+      log('error', error);
```

Comment:
and sometimes you're just passing through the error and not giving any other string...

Reply:
I think I did miss documenting this in Clay Log, but if you just give it an error then it'll format itself for an Error without you thinking about it. Since there wasn't an error message with this `catch` originally I didn't add one, but I can.

Reply:
Ahh, that'd be super handy to document